### ISSUE TITLE
`operationId`s containing `{` or `}` creates code which wont compile

### ISSUE THREAD

**Issue by aej** [aj-foster/open-api-generator#6]
I discovered this using the Hubspot companies API (https://api.hubspot.com/api-catalog-public/v1/apis/crm/v3/objects/companies)

In some cases the `operationId` contains `{...}`. Eg: `"delete-/crm/v3/objects/companies/{companyId}_archive"`

Open API generator will then generate code like

```elixir
  @spec delete__crm_v3_objects_companies_{company_id}_archive(String.t(), keyword) :: :ok | {:error, binary}
  def delete__crm_v3_objects_companies_{company_id}_archive(companyId, opts \\ []) do
...
```

which won't compile since its invalid elixir.

**Comment by aj-foster** [comment]
Hi @aej, thank you for this report! I have some code in #9 that should resolve it. If you have a chance, can you check and see if it works for you?

**Comment by aej** [comment]
Just tested it out and it works!

**Comment by aj-foster** [comment]
Resolved by #9. Thank you! ❤️
